### PR TITLE
CHORE - vf-forms linting

### PR DESCRIPTION
**Pull request by khawkins98**
A linting, code approach thing to namespace and use data attributes for the vf-forms js functionality.

### CODE REVIEW

**Comment by khawkins98** [comment]
There will be more to do here on the error messaging, but this does most of what's needed.

File: components/vf-form/vf-form__input/vf-form__input.scss [hunk 1]
```diff
@@ -139,37 +139,11 @@
...
+.vf-form__item.vf-form__floatlabel {
   position: relative;
 }
-.floatLabel {
+.vf-form__floatlabel {
   .vf-form__label {
     position: absolute;
   }
 }
-
```
Comment (sturobson) [inline components/vf-form/vf-form__input/vf-form__input.scss]: where's this gone?

Comment (khawkins98) [reply components/vf-form/vf-form__input/vf-form__input.scss]: Dropped this bit as it wasn't actually being used -- or is there a plan to use it?

Comment (sturobson) [reply components/vf-form/vf-form__input/vf-form__input.scss]: ah, that's true -- liberal copying and pasting happened here :D

Comment (khawkins98) [reply components/vf-form/vf-form__input/vf-form__input.scss]: hoarding :D

**Pull request merged**